### DISCUSSION TITLE
Fauxton/Futon to have a Copy Doc as a quick option to create entries

### DISCUSSION THREAD

**Discussion by DiegoJArg** [apache/couchdb#3039] [Ideas] [answered]
It currently have a button to copy entries contents into clipboard, and quick delete, but in order to create a new document:

- Copy some contents
- New Doc
- Append (not-replace) with Ctrl+V
- Delete old _id and _rev
- Add a colon "," to _id
- Do the changes that makes it a "different" document

**Comment by janl** [comment, marked as answer]
You can open a feature request ticket for Fauxton, or even better, a PR implementing this, here: http://github.com/apache/couchdb-fauxton/

**Reply by DiegoJArg** [reply]
Hi Jan Lehnardt, Wie geht ist Ihnen? thanks for your answer.
Or thanks for pointing to the right direction.

I noted that fauxton's git place doesn't have a "Discussions" Tab, only Issues reporting. 
But it is not an Issue, so probably someone would point me into another direction again.

I am failing to find where the community is mainly located at.
I get questions out of the learning curve but I can't find answers.
